src/pages/videoPlayer/videoPlayer.jsx

```diff
@@ -0,0 +1,132 @@
+import { useState } from "react";``
+import { useParams } from "react-router";
+import YouTube from "react-youtube";
+import { PlaylistModal } from "components";
+
+import { useAuth, useVideos, useUserResources, resourcesApiStateEnums } from "contexts";
+import { LoaderForComponent } from "components";
+import { useEffect } from "react";
+
+export function VideoPlayer() {
+  const { id } = useParams();
+  const { videos, isLoading } = useVideos();
+  const [currentVideo, setCurrentVideo] = useState({});
+  const [ showPlaylistModal, setShowPlaylistModal ] = useState(false)
+  const {
+    resourcesApiDispatch,
+    userResources: { likes: likedVideos, watchlater },
+  } = useUserResources();
+  const {
+    ADD_TO_LIKES,
+    ADD_TO_WATCHLATER,
+    REMOVE_FROM_WATCHLATER,
```

Comment:
Can be `REMOVE_FROM_WATCH_LATER`

Reply:
Thanks for the suggestion, I have fixed it in the latest commit